python/shotgun_desktop/initialization/shotgun.py

```diff
@@ -16,6 +16,9 @@
 
 from . import constants
 from distutils.version import LooseVersion
+from ..logger import get_logger
+
+logger = get_logger("initialization.shotgun")
```

Comment:
Actually, now that I think of it, not a good idea. The logging is initialized by bootstrap.py. We'd also have to update tk-framework-desktopserver to have everything be coherent. MIght as well wait for the work on the new Desktop to start and do everything at once in a clean fashion using the new API.